java/ql/src/Security/CWE/CWE-652/XQueryInjection.qhelp

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>The software uses external input to dynamically construct an XQuery expression used to retrieve data from an XML database, but it does not neutralize or incorrectly neutralizes that input. 
+This allows an attacker to control the structure of the query.</p>
+
+</overview>
+<recommendation>
+
+<p>Use parameterized queries. This will help ensure separation between data plane and control plane.</p>
+
+</recommendation>
+<example>
+
+<p>This example is a comparison of unused parameterized query and using parameterized query. 
+Parameterized query through <code>bindString</code>.</p>
+
+<sample src="XQueryInjection.java" />
+
+</example>
+<references>
+
+<li>cwe description: 
```

Comment:
```suggestion
<li>CWE description: 
```

Reply:
Isn't this reference unnecessary anyway?
AFAIK the reference to the CWE should be picked up automatically from the `.ql` file, right?

Reply:
Apparently this is true, didn't know that! Perhaps https://www.balisage.net/Proceedings/vol7/html/Vlist02/BalisageVol7-Vlist02.html is a good reference?